doc/python_api_reference_vDev.md

```diff
@@ -7659,9 +7688,18 @@ def cnot(
 # (in class stim.TableauSimulator)
 def copy(
     self,
+    *,
+    copy_rng: bool = False,
 ) -> stim.TableauSimulator:
     """Returns a copy of the simulator. A simulator with the same internal state.
 
+    Args:
+        copy_rng: If False, old and new simulators share the random number
```

Comment:
This seems backwards to me. If I ask for the rng to be copied it should be copied, not different.